.github/workflows/inductor-perf-test-nightly.yml

```diff
@@ -35,6 +35,11 @@ on:
         required: false
         type: boolean
         default: false
+      freezing_cudagraphs:
+        description: Run inductor_cudagraphs with freezing for inference?
+        required: false
+        type: boolean
+        default: false
```

Comment:
does this need to be on for it to run in the nightly @desertfire ?

Reply:
Yes, I guess you don't need it to run nightly?

Reply:
Nightly would be nice i guess. I'll test it out first non nightly and see that everything is working okay then go from there.